addons/forum-user-agent/addon.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Modify version information on forums",
+  "description": "Changes the text \"Flash version\" to \"Flash versions\" when clicking the globe icon on the forums.",
+  "info": [
+    {
+      "text": "This is useful for Scratch Addons developers to see if a bug that was reported on the forums was possibly caused by the extension.",
+      "id": "usefulness"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["editingScreens"]
+    }
+  ],
+  "tags": ["community", "forums"],
+  "versionAdded": "1.39.0",
```

Comment:
This will make the addon appear under "New addons". Should there be a note explaining that the new feature is the ability to disable it?